apps/vs-code-designer/src/app/utils/funcCoreTools/funcHostTask.ts

```diff
@@ -20,20 +20,20 @@ export interface IRunningFuncTask {
 
 export const runningFuncTaskMap: Map<vscode.WorkspaceFolder | vscode.TaskScope, IRunningFuncTask> = new Map();
 
+//TODO add new function which verifies SWA start and revert this one to original. call dynamically based off of whether swa init
 /**
- * Returns wheter the task is a func host start task.
+ * Returns whether the task is an SWA start task.
  * @param {vscode.Task} task - Function task.
- * @returns {number} Returns true if the task is a func host start task, otherwise returns false.
+ * @returns {boolean} Returns true if the task is an SWA start task, otherwise returns false.
  */
 export function isFuncHostTask(task: vscode.Task): boolean {
```

Comment:
For this I would suggest creating a different function rather than just patching this one.